internal/core/cobra_usage_builder.go

```diff
@@ -124,28 +125,40 @@ func buildExamples(cmd *Command) string {
 	for _, cmdExample := range cmd.Examples {
 		// Build title.
 		title := fmt.Sprintf("  %s", cmdExample.Short)
+		commandLine := ""
+		switch {
```

Comment:
maybe `case cmdExample.Raw != "" && case cmdExample.Raw != "" ` should trigger a warning

Nothing prevents a user to set both.

Reply:
Do you mean: `case cmdExample.Raw != "" && cmdExample.Request != ""` ?

It could be done in the generator validation and here keep the priority for the Raw example here ?

Reply:
could be done in the diagnostic tools